From 50a050e9e65d157a8903d0f78da06a60d863fd1c Mon Sep 17 00:00:00 2001
From: Srujana Challa <schalla@marvell.com>
Date: Tue, 27 Oct 2020 18:18:02 +0530
Subject: [PATCH 0927/1921] crypto: cn10k: start CPT PF driver version with 1

Start the CPT PF driver version from 1 for cn10k as the drivers are
differnet for 96XX and 106XX.

Signed-off-by: Srujana Challa <schalla@marvell.com>
Change-Id: I0995368aab994be214a6ac3f2af077ee55d09f8b
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/38652
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c b/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
index 0b76c77aef74..22da4c2242cb 100644
--- a/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
+++ b/drivers/crypto/marvell/cn10k/cn10k_cptpf_mbox.c
@@ -10,7 +10,7 @@
  * CPT PF driver version, It will be incremented by 1 for every feature
  * addition in CPT PF driver.
  */
-#define CN10K_CPT_PF_DRV_VERSION 0x2
+#define CN10K_CPT_PF_DRV_VERSION 0x1
 
 static void dump_mbox_msg(struct mbox_msghdr *msg, int size)
 {
-- 
2.31.1

